.github/workflows/unit-test.yml

```diff
@@ -67,4 +67,4 @@ jobs:
       run: make test
     
     - name: Publish test coverage
-      uses: codecov/codecov-action@v4
\ No newline at end of file
+      uses: codecov/codecov-action@v3
```

Comment:
Is downgrading intended?

Reply:
yes, v4 doesn't exists - it seems it is still in beta